contracts/funding/FundingOracleClient.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity ^0.5.0;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
+import "@chainlink/contracts/src/v0.5/ChainlinkClient.sol";
+
+
+/**
+ * @title Chainlink is an example contract which requests data from
+ * the Chainlink network
+ * @dev This contract is designed to work on multiple networks, including
+ * local test networks
+ */
+contract FundingOracleClient is ChainlinkClient, Ownable {
+  bytes32 public data;
+  uint8 public level;
+  string public url;
+  string public path;
+
+  /**
+   * @notice Deploy the contract with a specified address for the LINK
+   * and Oracle contract addresses
+   * @dev Sets the storage for the specified addresses
+   * @param _link The address of the LINK token contract
+   */
+  constructor(string memory _url, string memory _path, uint8 _level, address _link) public {
+    if (_link == address(0)) {
+      setPublicChainlinkToken();
+    } else {
+      setChainlinkToken(_link);
+    }
+
+    url = _url;
+    path = _path;
+    level = _level;
+    Ownable.initialize(msg.sender);
+  }
+
+  /**
+   * @notice Returns the address of the LINK token
+   * @dev This is the public implementation for chainlinkTokenAddress, which is
+   * an internal method of the ChainlinkClient contract
+   */
+  function getChainlinkToken() public view returns (address) {
+    return chainlinkTokenAddress();
+  }
+
+  /**
+   * @notice Creates a request to the specified Oracle contract address
+   * @dev This function ignores the stored Oracle contract address and
+   * will instead send the request to the address specified
+   * @param _oracle The Oracle contract address to send the request to
+   * @param _jobId The bytes32 JobID to be executed
+   */
+  function requestWinner(
+    address _pool,
+    address _oracle,
+    bytes32 _jobId,
+    uint256 _payment
+  )
+    public
+    onlyOwner
+    returns (bytes32 requestId)
+  {
+    Chainlink.Request memory req = buildChainlinkRequest(_jobId, address(this), this.fulfill.selector);
+    req.add("url", url);
+    req.add("copyPath", path);
+    req.addInt("level", level);
+    req.addBytes("pool", abi.encodePacked(_pool));
```

Comment:
here we are passing the pool address to the adapter, from what I tested it pads zeros to the right like: '0x123...0000000'.